File: Practica5/Marco teorico/2. Constructores/D. Propiedades de los constructores/D1.cpp
#include <iostream>
#include <string>


using namespace std;


class Persona {
private:
    string nombre;
    int edad;


public:
    // Constructor
    Persona(string n, int e) {
        nombre = n;
        edad = e;
    }


    // Método para mostrar los detalles de la persona
    void mostrarDetalles() {
        cout << "Nombre: " << nombre << ", Edad: " << edad << endl;
    }
};
int main() {
    // Crear un objeto Persona utilizando el constructor
    Persona persona1("Juan", 30);


    // Llamar al método para mostrar los detalles de la persona
    persona1.mostrarDetalles();


    return 0;
}

File: Practica5/Marco teorico/2. Constructores/D. Propiedades de los constructores/D1.java
public class Persona {
    private String nombre;
    private int edad;

    // Constructor
    public Persona(String n, int e) {
        nombre = n;
        edad = e;
    }

    // Método para mostrar los detalles de la persona
    public void mostrarDetalles() {
        System.out.println("Nombre: " + nombre + ", Edad: " + edad);
    }

    public static void main(String[] args) {
        // Crear un objeto Persona utilizando el constructor
        Persona persona1 = new Persona("Juan", 30);

        // Llamar al método para mostrar los detalles de la persona
        persona1.mostrarDetalles();
    }
}
